### ISSUE TITLE
jawn parser library incompatibility

### ISSUE THREAD

**Issue by NicolasRouquette** [muuki88/sbt-graphql#61]
With sbt 1.2.8, I'm running into a library conflict with jawn parser similar to this:
https://stackoverflow.com/questions/47227060/ammonite-classpath-clashes-with-github4s-java-lang-abstractmethoderror

Except for different URLs, the `build.sbt` is similar to the doc:

```
graphqlSchemas += GraphQLSchema(
  "sangria-example",
  "staging schema at http://try.sangria-graphql.org/graphql",
  Def.task(
    GraphQLSchemaLoader
      .fromIntrospection("http://try.sangria-graphql.org/graphql", streams.value.log)
      .loadSchema()
  ).taskValue
)
```

The stack trace is this:

```
[error] java.lang.AbstractMethodError
[error] 	at jawn.CharBasedParser.parseString(CharBasedParser.scala:90)
[error] 	at jawn.CharBasedParser.parseString$(CharBasedParser.scala:87)
[error] 	at jawn.StringParser.parseString(StringParser.scala:15)
[error] 	at jawn.Parser.rparse(Parser.scala:428)
[error] 	at jawn.Parser.parse(Parser.scala:338)
[error] 	at jawn.SyncParser.parse(SyncParser.scala:24)
[error] 	at jawn.SupportParser.$anonfun$parseFromString$1(SupportParser.scala:15)
[error] 	at jawn.SupportParser.parseFromString(SupportParser.scala:15)
[error] 	at jawn.SupportParser.parseFromString$(SupportParser.scala:14)
[error] 	at io.circe.jawn.CirceSupportParser$.parseFromString(CirceSupportParser.scala:7)
[error] 	at io.circe.jawn.JawnParser.parse(JawnParser.scala:16)
[error] 	at io.circe.parser.package$.parse(package.scala:8)
[error] 	at rocks.muki.graphql.schema.IntrospectSchemaLoader.introspect(SchemaLoader.scala:118)
[error] 	at rocks.muki.graphql.schema.IntrospectSchemaLoader.loadSchema(SchemaLoader.scala:85)
```

In Intellij, I see that under `External Libraries`, there is an entry for `sbt: sbt-and-plugins`.
In there, there are indeed two versions of jawn parser:

`jawn-parser_2.12-0.10.4.jar` (from SBT itself)
`jawn-parser_2.12-0.11.1.jar` (from `circe-jawn`)

It is unclear how to resolve this library conflict.

**Comment by NicolasRouquette** [comment]
I tried several workarounds in `project/plugins.sbt`:

```
libraryDependencies += "org.spire-math" %% "jawn-parser" % "0.10.4" force()
dependencyOverrides += "org.spire-math" %% "jawn-parser" % "0.10.4"
```

and:

```
libraryDependencies += "io.circe" %% "circe-jawn" % "0.9.3" exclude("org.spire-math", s"jawn-parser_${scalaBinaryVersion.value}")
```

Still the same error.

**Comment by muuki88** [comment]
There seems to be a [`circe-jackson`](https://github.com/circe/circe-jackson) module which we can use instead. WDYT?

**Comment by rtimush** [comment]
For the time being the workaround that worked for me was to add
```scala
dependencyOverrides ++= Seq(
  "io.circe" %% "circe-core" % "0.8.0",
  "io.circe" %% "circe-parser" % "0.8.0"
)
```
to my `plugins.sbt`. This might break some parts of `sbt-graphql`, but at least the basics work.

**Comment by muuki88** [comment]
Solved in version `0.13.0` thanks to @felixbr

**Comment by NicolasRouquette** [comment]
Great! Thanks @felixbr & @muuki88
